catroid/src/org/catrobat/catroid/ui/controller/LookController.java

```diff
@@ -313,6 +313,8 @@ private void copyImageToCatroid(String originalImagePath, Activity activity, Lis
 				imageName = oldFile.getName();
 			}
 
+			imageName = LookFragment.getFinalLookName(imageName,lookDataList);
```

Comment:
@thmq 
The [Jenkins build](https://jenkins.catrob.at/job/CatroidCheck/32) is [![green](https://jenkins.catrob.at/static/a6652cc0/images/16x16/blue.png)](https://jenkins.catrob.at/job/CatroidCheck/32) but running Checkstyle locally will report errors. e.g.

```
[ant:checkstyle] .../catroid/src/org/catrobat/catroid/ui/controller/LookController.java:316:56: warning: ',' is not followed by whitespace.
```